formatting-code/asparuh/JavaScript-code.js

```diff
@@ -0,0 +1,121 @@
+b=navigator.appName;
+
+addScroll=false;
+if ((navigator.userAgent.indexOf('MSIE 5')>0) || (navigator.userAgent.indexOf('MSIE 6'))>0) {

```

Comment:
Line character count - 96.
Unnecessary brackets.
Spacing between operands and operators.